public/app/AppWrapper.tsx

```diff
@@ -22,6 +22,13 @@ interface AppWrapperState {
   ngInjector: any;
 }
 
+/** Used by enterprise */
+let bodyRenderHooks: ComponentType[] = [];
+
+export function addBodyRenderHook(fn: ComponentType) {
```

Comment:
So this can be used for anything that needs to rendered outside of the Page?

Reply:
yes